tests/e2e/e2e_setup_test.go

```diff
@@ -18,8 +18,6 @@ const (
 	skipUpgradeEnv = "OSMOSIS_E2E_SKIP_UPGRADE"
 	// Environment variable name to skip the IBC tests
 	skipIBCEnv = "OSMOSIS_E2E_SKIP_IBC"
-	// Environment variable name to skip state sync testing
-	skipStateSyncEnv = "OSMOSIS_E2E_SKIP_STATE_SYNC"
```

Comment:
This is being used here, we should remove:
https://github.com/osmosis-labs/osmosis/blob/6e1b7a727b2a4ad5b44b962423494bb6826e91ca/tests/e2e/e2e_test.go#L1334

Reply:
hmmmm I thought it was used.  Any idea how toget one's editor to see that?  Or ci for that matter?

Reply:
Not going to block this PR on this change but we can replace the hardcoded value with `skipStateSyncEnv` in the future.

Thanks @faddat